Signal/src/call/CallService.swift

```diff
@@ -921,18 +923,28 @@ protocol CallServiceObserver: class {
     /**
      * The connection has been established. The clients can now communicate.
      */
-    internal func peerConnectionClientIceConnected(_ peerconnectionClient: PeerConnectionClient) {
+    internal func peerConnectionClientIceConnected(_ peerConnectionClient: PeerConnectionClient) {
         AssertIsOnMainThread()
 
+        if peerConnectionClient != self.peerConnectionClient {
```

Comment:
nit these should be `guard` statements to communicate to the ready we're anticipating early return